zerver/openapi/zulip.yaml

```diff
@@ -2009,8 +2009,9 @@ paths:
 
                                     - 0 = None. Used in events to indicate that the user no
```

Comment:
Another reason that's a good change is because this text is already within the documentation of an event, so "in events" is redundant.

I think the idea may have been to refer to the fact that a visibility policy of 0/None does not appear in the `user_topics` data in the /api/register-queue response. But really the place to cover that is in that response's own docs, and we do cover that there by simply not listing it as a possible value.